helix-term/src/commands/typed.rs

```diff
@@ -1803,6 +1831,13 @@ pub const TYPABLE_COMMAND_LIST: &[TypableCommand] = &[
             fun: force_buffer_close_all,
             completer: None,
         },
+        TypableCommand {
+            name: "buffer-goto",
+            aliases: &["b"],
+            doc: "Goto the buffer <i>.",
+            fun: goto_buffer_by_idx,
+            completer: None,
```

Comment:
It should be possible to add a small completer that only gives the current buffer numbers